pkg/settings/setting.go

```diff
@@ -123,6 +123,7 @@ var (
 
 	FleetMinVersion          = NewSetting("fleet-min-version", "")
 	RancherWebhookMinVersion = NewSetting("rancher-webhook-min-version", "")
+	CSPAdapterMinVersion     = NewSetting("csp-adapter-min-version", "")
```

Comment:
Not sure that I understand all of this comment - can you elaborate more?

This is grouped with the prior 3 settings because it is closest to them in terms of functionality (each controls the minimum applicable version). I'm ok with adding comments to explain the setting, but I definitely think that it should be grouped with similar settings rather than put in alphabetical order.